docs/ci.md

```diff
@@ -66,32 +63,38 @@ Here is an example of using the GitHub Web UI to launch an E2E workflow:
 
 The project currently supports the usage of the following runners for the E2E jobs:
 
-* [GitHub built-in GPU runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#specifications-for-gpu-larger-runners) -- referred to as `ubuntu-gpu` in our workflow files Only `e2e.yml` uses this runner.
-* Ephemeral GitHub runners launched on demand on AWS. Most workflows work this way, granting us access to a wider variety of infrastructure at lower cost.
+* Ephemeral GitHub runners launched on demand on AWS. All workflows currently work this way, granting us access to a wider variety of infrastructure at lower cost.
+
+We previously also used a  [GitHub built-in GPU runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#specifications-for-gpu-larger-runners) referred to as `ubuntu-gpu` -- this was deprecated due to cost.
```

Comment:
Same question about whether this context is helpful to future users.

Reply:
I think if we keep this section it's worth having just for historic context, but like I said above I'd also be fine to ax this whole section in the interest of keeping the doc a manageable length.